examples/dns/lib.rs

```diff
@@ -27,7 +27,7 @@ mod dns {
 
     /// Emitted whenever a new name is being registered.
     #[ink(event)]
-    struct Register {
+    pub struct Register {
```

Comment:
Do structs now have to be public when annotated with ` #[ink(event)]`?

Reply:
That's an interesting question actually: why would you think `#[ink(event)]` definitions should not required to be public?